src/TemplateLibrary/index.js

```diff
@@ -138,7 +138,7 @@ const TemplateLibraryComponent = (props) => {
           {props.addTemp
           && <NewClauseComponent addTempInput={props.addTemp} />}
         </Functionality>
-        <TemplateCards tempsHeight={libraryProps.TEMPLATES_HEIGHT} >
+        {filtered.length ? <TemplateCards tempsHeight={libraryProps.TEMPLATES_HEIGHT} >
```

Comment:
Can you change line 110 to `let filtered = templates || []` so that this line (141) does not error trying to read `filtered.length` if a user does not pass a templates prop (as it did in the test)

Reply:
got it, thanks @DianaLease !